README.md

```diff
@@ -1,10 +1,20 @@
 ## What is Dagger?
 
-Dagger is a CI/CD engine that is **programmable** and **container-native**.
+Dagger is a programmable CI/CD engine that runs your pipelines in containers.
 
-* **Programmable**: develop your CI/CD pipelines as code, in the same programming language as your application.
+### Programmable
 
-* **Container-native**: Dagger executes your pipelines entirely as [standard OCI containers](https://opencontainers.org), so the same pipeline can run on your local machine, a CI runner, a dedicated server, or any container hosting service. Other benefits of container-native pipelines include superior caching; cross-language instrumentation; compatibility with the massive Docker ecosystem; and more.
+Develop your CI/CD pipelines as code, in the same programming language as your application.
```

Comment:
It's just a suggestion but actually, it could also be your "favorite programming language". Imagine a Golang app but you wanna code your CI/CD pipeline in NodeJS or Python because it may be faster, it's also possible.

WDYT?

Reply:
Yeah both are equally good. I'm trying this one out of curiosity, but both are valid ways to pitch it.